docker.go

```diff
@@ -393,6 +393,23 @@ func (c *DockerContainer) ContainerIP(ctx context.Context) (string, error) {
 	return ip, nil
 }
 
+// ContainerIPs gets the IP addresses of all the networks within the container.
+func (c *DockerContainer) ContainerIPs(ctx context.Context) ([]string, error) {
+	ips := make([]string, 0)
+
+	inspect, err := c.inspectContainer(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	networks := inspect.NetworkSettings.Networks
+	for _, endpoints := range networks {
+		ips = append(ips, endpoints.IPAddress)
```

Comment:
Super NIT.

Because the `network` name is already used by an imported package, let's rename the loop variable:
```suggestion
	for _, nw := range networks {
		ips = append(ips, nw.IPAddress)
```

Reply:
Done